src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -460,6 +460,20 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
     DEBUG_LOG(PrintableModel(*nGraphFunc, "org_"));
 
     Transformations transformations(nGraphFunc, enableLPT, inferencePrecision, isLegacyAPI(), snippetsMode, engConfig);
+
+    if (!is_cpu_map_available()) {
+        ApplyPerformanceHints(config, nGraphFunc);
+    }
+
+    // update the props after the perf mode translated to configs
+    // TODO: Clarify the behavior of SetConfig method. Skip eng_config or not?
+    Config conf = engConfig;
+
+    conf.readProperties(config);
+    if (is_cpu_map_available()) {
+        GetPerformanceStreams(conf, nGraphFunc);
+    }
+
```

Comment:
This change can cause the number of streams is different from previous version.

Reply:
Thank you for reminding. This is recorded as a known issue in review.